packages/webdriverio/src/commands/browser/react$$.js

```diff
@@ -15,7 +15,8 @@
     });
  * </example>
  *
- * @alias browser.react$
+ * @alias browser.react$$
+ * @param {String=} selector React elements or HTML elements
```

Comment:
should it be `String|Function` like for common selector?

Reply:
no, this only takes a selector as string

Reply:
@CrispusDH It should be `String ` only, I am not sure what you meant by `Function `here for common selectors.